### ISSUE TITLE
halp can't compile :(

### ISSUE THREAD

**Issue by mew351** [WizardAim/WizardAim.github.io#2]
![image](https://cloud.githubusercontent.com/assets/12377381/26297916/ebe8ca3c-3e89-11e7-8968-610d9a5b760c.png)
Do you guys have a discord also?

**Comment by Malwarebites** [comment]
Yeah I'm currently working on the fix for it just realised I uploaded broken code so it needs to be changed anyway. should have it sorted by either tonight or tomorrow sorry about that.

**Comment by mew351** [comment]
Do you have a discord/any other way to contact you?
